orm/helpers.go

```diff
@@ -0,0 +1,70 @@
+package orm
+
+import (
+	"reflect"
+
+	"github.com/iov-one/weave/errors"
+)
+
+type limitedIterator struct {
+	// n is count of remaining elements
+	n int
```

Comment:
I think it would be better to call this variable `remaining` and remove the comment.